### PR TITLE
xlucas/go-ovh is deprecated, use ovh/go-ovh instead.

### PR DESCRIPTION
**Pull request by redochka**

### CODE REVIEW

**Comment by redochka** [comment]
I am unable to test that code is working and communication with OVH is working. I just made sure that code compiles. Sorry i am very new to golang and have trouble understanding why all imports are pointing to github/... instead of local packages...
Please test before merge.

**Comment by tlimoncelli** [comment]
I had to vendor the dependencies to get it to build on my system. Here's my PR that integrates what you did plus what I did: https://github.com/StackExchange/dnscontrol/pull/445

You can merge that into your branch (I can walk you through that) or I can merge my PR.

The one thing I wasn't able to do was to test this.  Our test procedure is pretty well documented.  It is Step 7 on this page: https://stackexchange.github.io/dnscontrol/writing-providers

The tl;dr version is that you'll want to set env variables with keys and domains and then run "go test". It will look something like this:

```
export OVH_APP_KEY=FILL_IN
export OVH_APP_SECRET_KEY=FILL_IN
export OVH_CONSUMER_KEY=FILL_IN
export OVH_DOMAIN=FILL_IN
cd integrationTest
go test -v -verbose -provider OVH
```

**Comment by patschi** [comment]
I think this is somehow stale. TBH hitting the merge button on #445 might be faster, wouldn't it be?

**Comment by tlimoncelli** [comment]
I've mered https://github.com/StackExchange/dnscontrol/pull/445

**Comment by patschi** [comment]
I think that can be closed now, @tlimoncelli? :)

**Comment by tlimoncelli** [comment]
Closing per @patschi 's advice.
